.travis.yml

```diff
@@ -155,6 +144,18 @@ jobs:
       rust: stable
       env: TASK=audit
 
+    - name: "lint Rust source using stable toolchain"
+      rust: stable
+      before_script:
+        - rustup component add clippy
+      env: TASK=clippy
+
+    - name: "lint Rust source using current development toolchain"
```

Comment:
Please move this one back up to where it came from.

Reply:
Resolved.